pkg/cmd/server/api/v1/conversions.go

```diff
@@ -19,6 +19,9 @@ func init() {
 			if obj.ServingInfo.RequestTimeoutSeconds == 0 {
 				obj.ServingInfo.RequestTimeoutSeconds = 60 * 60
 			}
+			if obj.ServingInfo.MaxRequestsInFlight == 0 {
+				obj.ServingInfo.MaxRequestsInFlight = 500
```

Comment:
ha, that number limits things which don't match this regex: 
```
longRunningRE := regexp.MustCompile("[.*\\/watch$][^\\/proxy.*]")
```

Turns out, that's not much: http://play.golang.org/p/oANtFn16cY